.github/workflows/config.yml

```diff
@@ -89,7 +89,7 @@ jobs:
 
       - name: Run buildstockbatch
         run: |
-          pip install git+https://github.com/NREL/buildstockbatch.git@develop
+          pip install git+https://github.com/NREL/buildstockbatch.git@bump-version
```

Comment:
Don't you want to revert it back to "develop" ?

Reply:
Yes, I'll do that after the companion PR is merged.